### DISCUSSION TITLE
Configure message timestamp format

### DISCUSSION THREAD

**Discussion by m-kay** [provectus/kafka-ui#2159] [Q&A] [answered]
Is it somehow possible to configure in which format the message timestamp is shown? Now the used format seems to be `MM.dd.yyyy HH:mm:ss` but I would like to have `dd.MM.yyyy HH:mm:ss`.

**Comment by Haarolean** [comment, marked as answer]
Hey, thanks for reaching out. I'll convert this into an issue, I have no clue why do we use that arbitrary retarded rollercoaster™ format by default. And we'll think of something to make it configurable for US users in the future.

**Reply by Haarolean** [reply]
#2162 #2163

**Reply by m-kay** [reply]
Great thank you. 

Just saw in the code that it is hardcoded: [Message.tsx#L69](https://github.com/provectus/kafka-ui/blob/master/kafka-ui-react-app/src/components/Topics/Topic/Details/Messages/Message.tsx#L69)
